Makefile

```diff
@@ -38,6 +38,9 @@ ifeq ($(TEST), true)
   CFLAGS   += $(TEST_FLAGS)
   CXXFLAGS += $(TEST_FLAGS)
   GPUFLAGS += $(TEST_FLAGS)
+  CFLAGS   += $(TEST_FLAGS)
```

Comment:
Accidental duplicates here?

Reply:
Yep. Fixing. 

There were a few spots like that that got duplicated in the rebase. I thought I got them but if you see any others please let me know.